pkg/destroy/aws/aws.go

```diff
@@ -1832,6 +1832,44 @@ func deleteS3(session *session.Session, arn arn.ARN, logger logrus.FieldLogger)
 	}
 	logger.Debug("Emptied")
 
+	var lastError error
+	var deleteMax int64 = 1000
+	err = client.ListObjectVersionsPagesWithContext(aws.BackgroundContext(), &s3.ListObjectVersionsInput{
+		Bucket:  aws.String(arn.Resource),
+		MaxKeys: &deleteMax,
```

Comment:
https://godoc.org/github.com/aws/aws-sdk-go/aws#Int64

Reply:
I think calling a function to pass a pointer is a bit silly. Is this normal practice?

Reply:
seems like it is for interacting with AWS SDK... an other cloud SDKs